packages/core/cypress.json

```diff
@@ -4,6 +4,7 @@
   "video": true,
   "trashAssetsBeforeRuns": true,
   "screenshotOnRunFailure": true,
+  "specPattern": ".faststore/cypress/integration/**/*.test.{js,jsx,ts,tsx}",
```

Comment:
does it affect the way we run the tests locally? because this folder will only exist when the cli run, right?

Reply:
@eduardoformiga How do we run the tests locally today? `yarn test` in the store root folder?

Reply:
yeap. 
`yarn test` opens cypress, and then we run triggering one `start` like button.